base_business_document_import/models/business_document_import.py

```diff
@@ -238,6 +238,36 @@ def _match_shipping_partner(self, shipping_dict, partner, chatter_msg):
         return partner
 
     @api.model
+    def _match_partner_bank(
+            self, partner, iban, bic, chatter_msg, create_if_not_found=False):
```

Comment:
But you can start with this one being more tolerant :wink: for reusing it in models without chatter.